mtags/src/main/scala-2/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -25,7 +25,7 @@ trait AutoImports { this: MetalsGlobal =>
 
   def doLocateImportContext(
       pos: Position,
-      autoImport: Option[AutoImportPosition]
+      autoImport: Option[AutoImportPosition] = None
```

Comment:
This is needed so that we only use auto import position when inserting actual imports:
```scala
package a
import java.nio.Paths
<<import_position>>

object A{
  class Path
  val a<<actual_position>>  = Paths.get(".")
}

```
view from the `actual_position` differs from `import_position`